### ISSUE TITLE
Toggle Series Button Present Below the Graph is not Announcing Position.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#17362]
#### Expected behaviour
Screen reader should announce the position for the toggle button.
Screen reader should announce as - 'Toggle series visibility chart region list, , Monthly Average Rainfall region list with 4 item Show Tokyo  toggle button pressed 1 of 4. 

#### Actual behaviour
Screen reader doesn't announce the position for the toggle button.
Screen reader is just announcing as - 'Toggle series visibility, Monthly Average Rainfall region list with 4 item Show Tokyo toggle button pressed. 

#### Steps to reproduce

1.Open URL: https://www.highcharts.com/demo/column-basic in the Edge browser
2.Navigate to ‘Basic Column’ under ‘Column and Bar Charts’ in the left navigation.
3. Enable NVDA.
4. Navigate to toggle present below the graph and observe the issue.

#### Product version

OS: Windows 11 22H2 (OS BUILD 25131.1000)
Edge browser: 102.0.1245.33

![image](https://user-images.githubusercontent.com/93735775/172264516-25d7a268-aa36-47ce-a5d1-4e8187e3a5f1.png)
